src/database/db_build.sql

```diff
@@ -0,0 +1,46 @@
+BEGIN ;
+DROP TABLE IF EXISTS countries CASCADE;
+DROP TABLE IF EXISTS teams CASCADE;
+
+CREATE TABLE countries(
```

Comment:
Please add a required field on some of these values e.g. team name

Reply:
@jema28 we add a required fields